### ISSUE TITLE
Allow testing against different Kafka Versions with Testcontainers

### ISSUE THREAD

**Issue by miguelpuyol** [akka/alpakka-kafka#783]
### Short description

Currently the trait `TestcontainersKafkaLike` fetches the latest Kafka version. However, not everyone is using the latest version.

### Details

Change the setup to allow a custom version as described in the Testcontainers [documentation](https://www.testcontainers.org/modules/kafka/).

Now:

```scala
 override def setUp(): Unit = {
    if (kafkaPortInternal == -1) {
      val kafkaContainer = new KafkaContainer()
      kafkaContainer.start()
      kafkaBootstrapServersInternal = kafkaContainer.getBootstrapServers
      kafkaPortInternal =
        kafkaBootstrapServersInternal.substring(kafkaBootstrapServersInternal.lastIndexOf(":") + 1).toInt
    }
    super.setUp()
  }
```

Possible solution:

```scala

val kafkaVersion: Option[String] = None

 override def setUp(): Unit = {
    if (kafkaPortInternal == -1) {
      val kafkaContainer = kafkaVersion match {
              case None => new KafkaContainer()
             case Some(version) => new KafkaContainer(version)
      }
      kafkaContainer.start()
      kafkaBootstrapServersInternal = kafkaContainer.getBootstrapServers
      kafkaPortInternal =
        kafkaBootstrapServersInternal.substring(kafkaBootstrapServersInternal.lastIndexOf(":") + 1).toInt
    }
    super.setUp()
  }
```

**Comment by miguelpuyol** [comment]
If you are ok, I can provide with a PR

**Comment by 2m** [comment]
Good suggestion! The default value could be derived using [sbt-buildinfo](https://github.com/sbt/sbt-buildinfo) from the version of Kafka client that is configured in the build.

**Comment by ennru** [comment]
The version number in there is the version of "Confluent Platform" not Kafka version https://hub.docker.com/r/confluentinc/cp-kafka/tags

**Comment by ennru** [comment]
This is even more relevant https://docs.confluent.io/current/installation/versions-interoperability.html

**Comment by ennru** [comment]
@miguelpuyol We would have loved your contribution, I just did it to make it into the upcoming patch release.

Fixed with #783.

**Comment by miguelpuyol** [comment]
No problem @ennru . I tried but I am not so familiar with `sbt-buildinfo` so I could not provide a good solution. Thanks for the quick feedback.
